query_prs_and_issues.graphql

```diff
@@ -0,0 +1,38 @@
+query prs_and_issues($want_prs: Boolean!, $pr_cursor: String, $want_issues: Boolean!, $issue_cursor: String) {

```

Comment:
So `pr_cursor` is sort of a meh `String`, but `want_prs` is very emphatically `Boolean`?

Reply:
Heh. This is how GraphQL marks required (`Meow!`) versus optional (`Meow`) parameters. The cursors are optional so they can be initially `null`, retrieving the first page of results.